docs/src/pages/style/color/ColorDemo.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Button from '@material-ui/core/Button';
+import IconButton from '@material-ui/core/IconButton';
+import MenuIcon from '@material-ui/icons/Menu';
+import Typography from '@material-ui/core/Typography';
+import AddIcon from '@material-ui/icons/Add';
+
+const styles = theme => ({
+  root: {
+    position: 'relative',
+    overflow: 'hidden',
+  },
+  appFrame: {
+    position: 'relative',
+    width: 260,
+    height: 360,
+    backgroundColor: theme.palette.background.paper,
+  },
+  statusBar: {
+    width: '100%',
+    height: 24,
+  },
+  menuButton: {
+    marginLeft: -12,
+    marginRight: 20,
+  },
+  fab: {
+    position: 'absolute',
+    bottom: theme.spacing.unit * 2,
+    right: theme.spacing.unit * 2,
+  },
+});
+
+const ColorDemo = props => {
+  const { classes, primary, theme, secondary } = props;
+  theme.palette.augmentColor(primary);
```

Comment:
Hum, this is not immutable.

Reply:
No, and it confused the heck out of me at first. It probably shouldn't have been added to the public API in that form (or written that way in the first place - my bad), but changing it now would be a breaking change.

Reply:
Yes, I screw up. We will sort that out in the next breaking change release. I'm keeping track of all the breaking changes we might introduce, I have 9 so far.